src/source/YamlGenerator.scala

```diff
@@ -148,7 +148,7 @@ class YamlGenerator(spec: Spec) extends Generator(spec) {
       "header" -> QuotedString(objcMarshal.include(td.ident)),
       "boxed" -> QuotedString(objcMarshal.boxedTypename(td)),
       "pointer" -> objcMarshal.isPointer(td),
-      "generic" -> false,
+      // "generic" -> false,
```

Comment:
what is this change for?  

should this commented out code just be removed?

Reply:
Djinni generates yaml for each user defined type it processes. Since we can't define generic interfaces in the IDL itself, the "generic" field being true or false does not really matter for IDL defined types.  This field is only useful in external types, which the user will provide custom marshallers. 

Historically Djinni writes "generic" -> true in the JNI section, and nothing in the other sections.  Since we have added generic support for ObjC, we do have this field in objc types, so I wrote it out in the yaml too.  But writing it out explicitly causes a lot of the existing yaml files to be modified even though the new field merely has the default value,  which does not impact codegen at all.

Here I choose to omit the field in the yaml generator (and in the case of JNI, keep it true) when it does not impact codegen, to minimize the generated code change.